src/style.css

```diff
@@ -0,0 +1,103 @@
+@font-face {
+  font-family: "Avenir";
+  src: local("Avenir"),
+    url("fonts/Avenir-Book.ttf") format("ttf"),
+    url("fonts/Avenir-Heavy.ttf") format("ttf"),
+    url("fonts/Avenir.ttc") format("ttc");
+}
+
+* {
+  font-family: "Avenir-Book", sans-serif;
+  font-weight: 300;
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+form {
+  margin-left: 15px;
+}
+
+.big-bar[data-qa] {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  height: 70px;
+  padding: 0;
+  border: 1px solid #e1e7ed;
```

Comment:
Move all repeatable colors to the variables.

[More info](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)